### ISSUE TITLE
How to pass PDFLaTeX options?

### ISSUE THREAD

**Issue by dpo** [ftilmann/latexdiff#233]
My document must be compiled with `-shell-escape`. How do I instruct `latexdiff` to pass that flag to `pdflatex`?

Thanks in advance!

**Comment by ftilmann** [comment]
I guess you refer to  `latexdiff-vc`.
This is supposed to work like this: 
```
 latexdiff-vc - --pdf -c LATEX='pdflatex --shell-escape' ... 
```
but there is a bug in the current release version that makes it impossible to include space. It's fixed in commit  dbf247f  , but if you want to patch your local version, you can also simply find the line
```
  $CFG{LATEX} = "pdflatex";
```
in latexdiff-vc and add the desired options.

**Comment by dpo** [comment]
Thank you!
